articles/clients/connections.md

```diff
@@ -6,37 +6,24 @@ crews: crew-2
 
-## Example configuration
-
-Let's suppose that you want to implement this architecture.
+## Example multi-tenant configuration
 
-![Client connections example](/media/articles/applications/applications-connections-example.png)
+If you have two separate domains (for example, public facing and internal), or two groups of connections you'd like to allow users, the best solution is to create a second Auth0 tenant via the settings menu in the top right on the [Dashboard](${manage_url}). This will allow you to have separate sets of users, clients, and connections for the two groups of users and applications you need to support.
 
-You have two applications: a timesheets application and a customer portal. Users should login to the timesheets application either using their Active Directory credentials or their Google apps social connection. The customer portal on the other hand should be accessible via ADFS, Azure AD, Google Apps or LinkedIn authentication.
+Let's suppose that you have two applications: an internal timesheets application and a customer portal. Users should login to the timesheets application either using their Active Directory credentials or their Google apps social connection. The customer portal on the other hand should be accessible via Facebook, Google, or LinkedIn authentication.
 
 You can configure this in Auth0 as follows:
-- Create a client for the timesheets application: `Fabrikam Employee Timesheets`.
-- Create a client for the customer portal: `Fabrikam Customer Portal`.
-- Configure the following four [Enterprise connections](${manage_url}/#/connections/enterprise): Active Directory / LDAP, ADFS, Microsoft Azure AD and Google Apps.
-- Configure the following [Social connection](${manage_url}/#/connections/social): LinkedIn.
-- For the `Fabrikam Employee Timesheets` client enable the Active Directory / LDAP and Google Apps connections.
-- For the `Fabrikam Customer Portal` client enable the ADFS, Microsoft Azure AD, Google Apps and LinkedIn connections.
-- That's it!
+
+- Create a tenant `Fabrikam-Internal` for your internal domain, and a client within it `Fabrikam Employee Timesheets` for timesheets.
+- Create a second tenant `Fabrikam-Public` for your public-facing domain, and a client within it `Fabrikam Customer Portal` for the customer portal.
+- Configure the following [Enterprise connections](${manage_url}/#/connections/enterprise) for the `Fabrikam-Internal` tenant: Active Directory / LDAP, and Google Apps; once each is set up, check the **Clients** tab to enable them.
+- Configure the following [Enterprise connection](${manage_url}/#/connections/enterprise) for the `Fabrikam-Public` tenant: Google Apps; once it is set up, check the **Clients** tab to enable it.
+- Configure the following [Social connections](${manage_url}/#/connections/social) for the `Fabrikam-Public` tenant: Facebook, LinkedIn; once each is set up, check the **Clients** tab to enable it.
```

Comment:
Here we say to use the Clients tab to enable/disable a connection but we removed the guidance that says how to do this earlier in the doc. Seems to me that the guidance at the beginning should stay as it was, I don't see the point in removing it unless the screen will be removed.

Reply:
Responded on Slack... TL;DR is that those were are separate places, and unfortunately, with the dashboard UI as it is currently, you have to toggle the clients on. The idea is that you should toggle them all on, I guess.